### PR TITLE
Delegated render of LinksTo, LinksToMany field should use fitted format by default

### PR DESCRIPTION
**Pull request by lukemelia**
- it should be rendered in a container that has height and width
- Use "Strip" format, height: 58px, width: 100% of available space
- Make <@fields.xxxx … /> invocation support splattributes to allow sizing of container

Note that overlays will hopefully work better with these size fitted formats after #1496

### CODE REVIEW

File: packages/base/field-component.gts [hunk 1]
```diff
@@ -282,7 +308,7 @@ export function getBoxComponent(
           works if we use up all the space horizontally and vertically that is available
           to the card since some of our queries are height queries
         */
-        height: 100%;
+        height: 58px;
```
Comment (backspace) [inline packages/base/field-component.gts:311]: The comment above is still about `100%'.

File: packages/base/field-component.gts [hunk 2]
```diff
@@ -331,16 +357,16 @@ export function getBoxComponent(
...
+function defaultFieldFormats(containingFormat: Format): FieldFormats {
+  switch (containingFormat) {
     case 'edit':
-      return 'edit';
+      return { fieldDef: 'edit', cardDef: 'edit' };
     case 'isolated':
     case 'fitted':
     case 'embedded':
-      return 'embedded';
+      return { fieldDef: 'embedded', cardDef: 'fitted' };
```
Comment (backspace) [inline packages/base/field-component.gts:367]: For my own understanding, why are these different?

File: packages/base/links-to-many-component.gts [hunk 3]
```diff
@@ -358,7 +367,7 @@ function getEditorChildFormat(
   ) {
     return 'atom';
   }
-  return 'embedded';
+  return 'embedded'; // TODO: correct?
```
Comment (backspace) [inline packages/base/links-to-many-component.gts]: I unfortunately can’t help confirm this but hopefully someone can!

File: packages/runtime-common/constants.ts [hunk 4]
```diff
@@ -26,7 +26,7 @@ export const primitive = Symbol('cardstack-primitive');
 export const aiBotUsername = 'aibot';
 
 export const CardContextName = 'card-context';
-export const DefaultFormatContextName = 'default-format-context';
+export const DefaultFormatsContextName = 'default-format-context';
```
Comment (backspace) [inline packages/runtime-common/constants.ts:29]: Should it be plural within the string too?

File: packages/base/links-to-many-component.gts (hunk 3 above)
Comment (tintinthong) [inline packages/base/links-to-many-component.gts]: Stale comment?

**Review by tintinthong — APPROVED** [review]

**Review by backspace — COMMENTED** [review]
wow I had this written but failed to submit, sorry 😞

**Pull request merged**